### PR TITLE
add TSSOP-24-1EP

### PR DESCRIPTION
**Pull request by dudelectric**
Push of 
**TSSOP-24-1EP_4.4x7.8mm_P0.65mm_EP3.2x5mm** footprint 
for part **LED1642GWXTTR**

Part Datasheet: https://www.st.com/resource/en/datasheet/led1642gw.pdf

### CODE REVIEW

File: scripts/Packages/Package_Gullwing__QFP_SOIC_SO/size_definitions/tssop.yaml [hunk 1]
```diff
@@ -517,6 +517,27 @@ TSSOP-24_4.4x7.8mm_P0.65mm:
   num_pins_x: 0
   num_pins_y: 12
 
+TSSOP-24-1EP_4.4x7.8mm_P0.65mm_EP3.2x5mm:
+  size_source: 'JEDEC MO-153 Var AD https://www.jedec.org/document_search?search_api_views_fulltext=MO-153'
```
Comment (chschlue) [inline scripts/Packages/Package_Gullwing__QFP_SOIC_SO/size_definitions/tssop.yaml:521]: This package is underspecified by this size source, MO-153 only specifies minimum EP size.
Please use ST documentation.

File: scripts/Packages/Package_Gullwing__QFP_SOIC_SO/size_definitions/tssop.yaml [hunk 2]
```diff
@@ -517,6 +517,27 @@ TSSOP-24_4.4x7.8mm_P0.65mm:
...
+  body_size_x: 4.4 +/-0.1
+  body_size_y: 7.8 +/-0.1
```
Comment (chschlue) [inline scripts/Packages/Package_Gullwing__QFP_SOIC_SO/size_definitions/tssop.yaml:523]: Please use min..nom..max for easier comparison with the datasheet.

File: scripts/Packages/Package_Gullwing__QFP_SOIC_SO/size_definitions/tssop.yaml [hunk 3]
```diff
@@ -517,6 +517,27 @@ TSSOP-24_4.4x7.8mm_P0.65mm:
...
+  overall_size_x: 6.4
```
Comment (chschlue) [inline scripts/Packages/Package_Gullwing__QFP_SOIC_SO/size_definitions/tssop.yaml:524]: Add tolerance

File: scripts/Packages/Package_Gullwing__QFP_SOIC_SO/size_definitions/tssop.yaml [hunk 4]
```diff
@@ -517,6 +517,27 @@ TSSOP-24_4.4x7.8mm_P0.65mm:
...
+  body_height: 1.2
+  lead_width: 0.17 .. 0.30
```
Comment (chschlue) [inline scripts/Packages/Package_Gullwing__QFP_SOIC_SO/size_definitions/tssop.yaml:526]: 0.19 .. 0.30

File: scripts/Packages/Package_Gullwing__QFP_SOIC_SO/size_definitions/tssop.yaml [hunk 5]
```diff
@@ -517,6 +517,27 @@ TSSOP-24_4.4x7.8mm_P0.65mm:
...
+  lead_len: 0.6 +/-0.15
```
Comment (chschlue) [inline scripts/Packages/Package_Gullwing__QFP_SOIC_SO/size_definitions/tssop.yaml:527]: See above for tolerance

File: scripts/Packages/Package_Gullwing__QFP_SOIC_SO/size_definitions/tssop.yaml [hunk 6]
```diff
@@ -517,6 +517,27 @@ TSSOP-24_4.4x7.8mm_P0.65mm:
...
+  num_pins_y: 12
+  EP_size_x:
+    minimum: 3
+    nominal: 3.2
+    maximum: 3.4
+  EP_size_y:
+    minimum: 4.8
+    nominal: 5
+    maximum: 5.2
+  EP_num_paste_pads: [1, 1]
```
Comment (chschlue) [inline scripts/Packages/Package_Gullwing__QFP_SOIC_SO/size_definitions/tssop.yaml:539]: This being a relatively large EP, please split into more paste pads (like [2, 3] or possibly [3, 4])

**Review by chschlue — CHANGES_REQUESTED** [review]

File: scripts/Packages/Package_Gullwing__QFP_SOIC_SO/size_definitions/tssop.yaml (hunk 6 above)
Comment (dudelectric) [reply scripts/Packages/Package_Gullwing__QFP_SOIC_SO/size_definitions/tssop.yaml]: Hi @chschlue Thank you for the review, Just commited changes.
Thanks

**Review by chschlue — APPROVED** [review]
Thanks.

**Pull request merged**